packages/components/label/src/label.tsx

```diff
@@ -47,7 +47,9 @@ type TLabelProps = {
    * <br />
    * This is required when `children` is `undefined` and vice versa
    */
-  intlMessage?: MessageDescriptor;
+  intlMessage?: MessageDescriptor & {
+    values?: Record<string, React.ReactNode>;
```

Comment:
ReactNode seem to be already imported from react in this file, same as other places with similar feedback. Maybe we just need to add `type` to all places where they have been imported. 
Suggestion for taking modifying the `React.ReactNode` syntax is just for consistency with other files. 
```suggestion
    values?: Record<string, ReactNode>;
```

Reply:
Yes, you're right but I this (also `ContentNotification` and `Link`) were out of the scope of this PR. I just modified its `MessageDescriptor` to allow parameterized messages and din't look into them any further.

Anyway, I can also make these changes now that I'm modifying those files for the other reason.

Thanks 👍

Reply:
Ah, I see. Thanks for clarifying.

Reply:
@ddouglasz I standarized `ReactNode` usage in this commit: 7f0fc7356c510a2b6754da069cd5b6489114d228

Reply:
Great!